monero-outreach-docs/README_i18n.md

```diff
@@ -6,33 +6,45 @@ Welcome to the Monero Outreach Localization. This directory contains all the fin
 
 Our goal as a community is to create infrastructure and tools that can be used as an information bridge between core developers, community members, and the general public. We aim to ensure the public is informed of the many benefits of privacy and fungibility in cryptocurrencies.
 
+## For XMR Rewards
+
+Visit: [Taiga EPIC #69 Monero Outreach Localization](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/69).
+
 
 ### Contributing
 
+Please review the translations tips (standardized terms) we expect everyone to follow:https://github.com/erciccione/monero-translations/blob/master/translation-tips.md.
```

Comment:
This point to my personal repo. It should point to https://github.com/monero-ecosystem/monero-translations/blob/master/translation-tips.md

Reply:
Ok, I will be fixing it. Thank you.